llvm/test/tools/llvm-cas-object-format/Inputs/print-out-flatv1.txt

```diff
@@ -1,13 +1,16 @@
 Triple: x86_64-apple-darwin
 
 SECTION: __TEXT,__text | MemProt: R-X
+CASID: llvmcas://2becf1eca41baad177b5d7a795d3db38d60b1ce1
```

Comment:
as mentioned in the meeting, this hash is not stable since the input is generated from `.ll` file